apps/palette-generator/tsconfig.spec.json

```
@@ -0,0 +1,25 @@
+{
+  "extends": "../../tsconfig.jest",
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "strictNullChecks": false,
+    "strictPropertyInitialization":false,
+    "noImplicitAny": false,
+    "incremental": true,
+    "composite": true,
+    "declarationMap": false,
+    "strict": false,
```

Is part of them already included in parent ?
It sounds like you require only

```suggestion
    "esModuleInterop": true,
    "incremental": true,
    "composite": true,
```